src/convertToObject.js

```diff
@@ -7,6 +7,19 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  return sourceString
+    .split(';')
+    .map((entry) => entry.trim())
+    .filter((entry) => entry)
+    .reduce((cssProperties, entry) => {
+      const [key, value] = entry
+        .split(':')
+        .map((spltEntry) => spltEntry.trim());
+
+      cssProperties[key] = value;
```

Comment:
When assigning the value to the `cssProperties` object, ensure that you handle the case where the property name is empty or invalid, as the current implementation would still add such entries to the object.